### DISCUSSION TITLE
How to create new context in the faststore.starter

### DISCUSSION THREAD

**Discussion by evander-corebiz** [vtex/faststore#2177] [General]
Hello, how are you? We're building a new store using the last version of faststore.starter and we need to create a new context. I don't find a way to do this. We only can override components and sections. I'm thinking to use Redux, but i didn't want to use a new package. There is another way to do this?

**Comment by hellofanny** [comment]
Helloo @evander-corebiz, I hope you're doing fine. I'd like to understand why a new context is needed. Is there a specific use case for it? Currently, FastStore V2 does not support context override 💭

**Comment by evander-corebiz** [comment]
Hello, thanks for the answer. We needed a new context to create a wishlist and use it across all pages. We have a wishlist products page and we needed show a button inside of all products card. We did it, but with a new context it's would be easier.
